website/templates/project/settings.mako

```diff
@@ -316,46 +316,40 @@
-                                <i>None</i>
-                                <!-- /ko -->
+                        <!-- ko if: affiliatedInstitutions().length > 0 -->
+                        <label>Affiliated Institutions: </label>
+                        <!-- /ko -->
                         <table class="table">
                             <tbody>
                                 <!-- ko foreach: {data: affiliatedInstitutions, as: 'item'} -->
                                 <tr>
                                     <td><img class="img-circle" width="50px" height="50px" data-bind="attr: {src: item.logo_path}"></td>
                                     <td><span data-bind="text: item.name"></span></td>
                                     % if 'admin' in user['permissions']:
-                                        <td><a data-bind="click: $parent.clearInst"><i class="fa fa-times"></i></a></td>
+                                        <td><button data-bind="click: $parent.clearInst" class="pull-right btn btn-danger">Remove</button></td>
                                     % endif
                                 </tr>
                                 <!-- /ko -->
 
                             </tbody>
                         </table>
-                        % if 'admin' in user['permissions']:
-                            <div data-bind="visible: hasThingsToAdd">
-                                <a data-bind="click: toggle">Add institution</a>
-                                <div data-bind="visible: showAdd()">
-                                    <div class="help-block">
-                                        You are authorized to affiliate your projects with the following institutions:
-                                    </div>
-                                    <div class="radio">
-                                        <div data-bind="foreach: {data: availableInstitutions, as: 'item'}">
-                                            <div>
-                                            <label>
-                                                <input type="radio" data-bind="value: item.id, checked: $parent.selectedInstitution">
-                                                <p data-bind="text: item.name"></p>
-                                            </label>
-
-                                            </div>
-                                        </div>
-                                    </div>
-                                    <button data-bind="click: submitInst, css: {disabled: selectedInstitution() == null}" class="btn btn-success">Affiliate</button>
-                                </div>
-                           </div>
-                        % endif
+                            </br>
+                        <!-- ko if: availableInstitutions().length > 0 -->
+                        <label>Available Institutions: </label>
+                        <!-- /ko -->
```

Comment:
I *think* that the institutions should show without the ability to change them.  @saradbowman is that correct?